Formula/gnunet.rb

```diff
@@ -32,7 +32,7 @@ class Gnunet < Formula
 
   def install
     ENV.deparallelize if OS.linux?
-    system "./configure", "--prefix=#{prefix}"
+    system "./configure", "--prefix=#{prefix}", "--with-microhttpd"
```

Comment:
Odd behavior since they list this as an optional package (defaulting to disabled) but have a mandatory dependency:
```
Optional Packages:
  --with-PACKAGE[=ARG]    use PACKAGE [ARG=yes]
  --without-PACKAGE       do not use PACKAGE (same as --with-PACKAGE=no)
  ...
  --with-microhttpd=PFX   base of libmicrohttpd installation
```

```
AS_IF([test "x$libmhd" = "x0"],
  [AC_MSG_ERROR([GNUnet requires libmicrohttpd >= 0.9.63])])
```

On side note, setting a `=PFX` path value doesn't work since they reset `PKG_CONFIG_PATH`, which causes `pkg-config` to complain about not finding `gnutls`.

Only `--with-microhttpd` or `--with-microhttpd=yes` works as uses `pkg-config` with our default `PKG_CONFIG_PATH`

Reply:
maybe we should raise this up and confirm if it got changed to mandatory dependency?

Reply:
The default is automatic, but that automatic tampers with `PKG_CONFIG_PATH`.

This is generally better anyway rather than relying on automatic, since if a dependency check fails then it doesn't silently skip it.